src/components/layout/Nav.jsx

```diff
@@ -207,6 +210,12 @@ export default function Nav({
       >
         <Feedback link={config.feedbackLink} setFeedback={setFeedback} />
       </Dialog>
+      <Dialog
+        open={resetfilters}
+        maxWidth={isMobile ? 'sm' : 'xs'}
+      >
+        <ResetFilters link={config.resetfiltersLink} setResetFilters={setResetFilters} />
```

Comment:
It doesn't look like you're using `link={config.resetfiltersLink} ` anywhere in the `ResetFilters.jsx` dialog? Did you forget to add something? (If not, you can just remove it entirely)